gulpfile.js

```diff
@@ -218,33 +239,37 @@ gulp.task('build-frontend-games-html', ['clean-frontend'], function () {
 
 });
 
-gulp.task('build-frontend-js-vendor', ['clean-frontend'], function () {
-  
-  var b = browserify({
-    entries: buildConfig.frontend.js.vendor.src,
-    debug: true
-  });
+function buildBrowserifyBundle(bundleOptions) {
```

Comment:
The `buildBrowserifyBundle()` does the heavy lifting in building the _JavaScript_, @greybubble.